test/re_test.cc

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 #include <gtest/gtest.h>
-#include "re.h"
+#include "../src/re.h"
```

Comment:
this is unfortunate - should we add the src/ folder to the test/CMakeLists.txt include_directories instead?